source/css/_objects.carousels.scss

```diff
@@ -50,33 +81,56 @@
 }
 
 /**
- * Carousel with Text
+ * Carousel with Text Inset
  */
-.c-carousel__item-text__wrap {
-  color: $c-white;
-  position: absolute;
-  left: 0;
-  height: 100%;
-  width: 100%;
-  top: 0;
+.c-carousel--inset {
+  .c-carousel__item--inset {
+    @media (min-width: 650px) {
+      max-height: 40rem;
+    }
+  }
 }
 
-.c-carousel__item-text {
-  position: relative;
-  height: 100%;
-  width: 100%;
-  padding-bottom: 0;
-  display: flex;
-  justify-content: center;
-  align-items: flex-start;
-  flex-direction: column;
+.c-carousel__item--inset {
+  font-size: 0.5em;
 
-  &--inner {
+  img {
     width: 100%;
-    display: flex;
-    justify-content: flex-start;
-    flex-direction: column;
-    align-items: flex-start;
+  }
+
+  .c-carousel__item-text__wrap {
+    color: #fff;
+    position: relative;
+    width: 100%;
+    z-index: 100;
+
+    @media (min-width: 650px) {
```

Comment:
To be consistent with the current breakpoints, I would use `@include ('>medium') {}` or `@include('>large') {}` ect., so all the breakpoints match.